cockroachdb/templates/statefulset.yaml

```diff
@@ -65,6 +65,17 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
+        {{- if .Values.statefulset.securityContext.enabled }}
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+            privileged: false
+            readOnlyRootFilesystem: true
+            seccompProfile:
+              type: "RuntimeDefault"
```

Comment:
This seccompProfile should be moved to the pod securityContext instead of keeping it in container securityContext which will be consistent to all other jobs securityContext